upgrade/scripts/k8s/rr_critical_service_restart.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/python3
```

Comment:
If you're going to bother to change this with the PR, it would be just as simple to set the file as executable instead, to make it easier for users to call it. Up to you, but it seems silly to remove this rather than just make the file executable.